examples/hello_world.rs

```diff
@@ -0,0 +1,88 @@
+#![cfg_attr(windows, feature(abi_vectorcall))]
+use ext_php_rs::{constant::IntoConst, prelude::*, types::ZendClassObject};
+
+#[derive(Debug)]
+#[php_class]
+pub struct TestClass {
+    #[prop]
```

Comment:
i find this name not really consistent given other macros, maybe it would be nice to have a better one.

(Not necessarily in this PR, but before the next release at least, we can profit from this BC break to make things consistent ?)

Reply:
So your suggestion would be something like `#[php_prop]`?

Not a 100% sure how I feel about that given that it isn't an actual macro and its simpler the way it is. On the other hand a user might not actually care about that distinction.

I'll open an issue for this.

Reply:
#391

Reply:
Maybe something like serde is doing would be a good fit.

`#[php(prop)]`
`#[php(rename = "foo")]`
[...]

Reply:
Yeah, i was thinking at something like this, not sur how doable it is since i'm not used to write derive macro

Reply:
Should be doable.

Ill resolve these conversations and add the suggestion to the issue.